src/utils/url.ts

```diff
@@ -50,10 +50,8 @@ export const openUrl = (url: string) => {
         event.addMetadata("utils/url::openUrl", { url });
       }
     );
-    // @debt keep the checking in place so we can debug further, but don't block attempts to open
-    // return;
+    return;
   }
-
```

Comment:
[nit] We want to keep this whitespace as it provides clean separation from the if statement below, etc.

Reply:
fixed